src/libraries/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -119,6 +119,7 @@
     <Compile Include="System\PseudoCustomAttributeTests.cs" />
     <Compile Include="System\RangeTests.cs" />
     <Compile Include="System\RankExceptionTests.cs" />
+    <Compile Include="System\Runtime\Versioning\RequiresPreviewFeaturesAttributeTests.cs" />
```

Comment:
(attaching the comment to the right line)

Nit: It would be nice to put this into the existing block of System\Runtime... files